### ISSUE TITLE
P5 regexes blow up all highlighting

### ISSUE THREAD

**Issue by zoffixznet** [MadcapJake/language-perl6fe#8]
Hey,

Perl 6 allows use of Perl 5 regexes by using `m:P5` modifier. Unfortunately, the highlighter still attempts to parse them as P6 regexes, with some rather dramatic consequences. In the picture below, you can see how the single quote inside the regex makes all the code that follows the regex to be mis-highlighted.

I suspect making the highlighter correctly highlight Perl 5 regexes might be a lot of work, so I think making it NOT highlight `m:P5` regexes as Perl 6 regexes would be a good-enough fix, since at least nothing would be broken.

Here's the picture:

![bug1](https://cloud.githubusercontent.com/assets/5747918/11847737/dffb7a8c-a3ec-11e5-9222-9852fcb926a7.png)

And here's the actual code:

``` perl6
my $ATTR_RE = m:P5/(?s:(["'])(.*?)\g{-2}|([^>\s]*))/x;

my $horribly-mishighlighted-code = ':(';
```

**Comment by MadcapJake** [comment]
Yeah I think the best solution for this might be to just print it as regular text.  Is it always the last/only adverb?

**Comment by zoffixznet** [comment]
Not sure, but probably the answer is yes, since some adverbs don't make sence in P5 regexes.

**Comment by MadcapJake** [comment]
I'm almost finished with this, but it won't support the `/x` as I am getting errors in the repl that this is unsupported.  So I will need to allow other adverbs then.

**Comment by zoffixznet** [comment]
Oh, right, the `/x` is just left over bogus code, sorry.

**Comment by MadcapJake** [comment]
No problem! Almost done!

**Comment by MadcapJake** [comment]
Fixed in v1.4.0
